sql-files/upgrades/2020-08-15--13-00.sql

```diff
@@ -0,0 +1,30 @@
+#1597467600
+
+-- This file is part of Hercules.
+-- http://herc.ws - http://github.com/HerculesWS/Hercules
+--
+-- Copyright (C) 2019-2020 Hercules Dev Team
+--
+-- Hercules is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-- Add attendance data to accounta_data table
+ALTER TABLE `account_data` ADD COLUMN `attendance_count` TINYINT(3) UNSIGNED NOT NULL DEFAULT '0' AFTER `base_death`;
+ALTER TABLE `account_data` ADD COLUMN `attendance_timer` BIGINT(20) NULL DEFAULT '0' AFTER `attendance_count`;
+
+-- Delete column for attendance at char table
+ALTER TABLE `char` DROP COLUMN `attendance_count`;
```

Comment:
Would it make sense to update the new `account_data` fields with the nonzero values from the `char` table, so that existing data isn't lost?

Reply:
@MishimaHaruna You mean getting the current values from the char table then insert into the account data table? Hmm, i tried this before making this PR, but I'm getting problem as account_data table not always have values saved in it unless bank or the mod exp was given value, then in char table wherein I should just get the highest value for attendance count and attendance timer should just be copied only to be sent into the account_data table

Reply:
Have you tried with a `REPLACE ... SELECT` query?  Something similar to:

```
REPLACE INTO `account_data` (`account_id`, `attendance_count`, `attendance_timer`)
SELECT `account_id`, MAX(`attendance_count`), MAX(`attendance_timer`)
FROM `char`
WHERE `attendance_count` <> 0 AND `attendance_timer` <> 0
GROUP BY `account_id`
```

Reply:
I updated the migration to include this